sklearn/preprocessing/tests/test_function_transformer.py

```diff
@@ -81,3 +81,37 @@ def test_np_log():
         FunctionTransformer(np.log1p).transform(X),
         np.log1p(X),
     )
+
+
+def test_kw_arg():
+    X = np.linspace(0, 1, num=10).reshape((5, 2))
+
+    F = FunctionTransformer(np.around, kw_args=dict(decimals=3))
+
+    # Test that rounding is correct
+    np.testing.assert_array_equal(F.transform(X),
```

Comment:
For consistency, I copied that from the existing ``test_np_log`` test.

Do you want me to change it there as well?